components/channel_view/channel_view.jsx

```diff
@@ -53,7 +53,7 @@ export default class ChannelView extends React.PureComponent {
     componentDidMount() {
         const platform = window.navigator.platform;
 
-        $('body').addClass('app__body');
+        document.body.classList.add('app__body', 'channel-view');
```

Comment:
Also, what's the reason we have to add a class like this and can't add it to the `app_body` node directly?

Reply:
ahh just read your description, i see why we have to do this.

Reply:
I intentionally kept the changes minimal because there were no existing tests for the file and i kinda was lazy to add them for a small change as this.

Reply:
@ali-farooq0 Removed jquery form the file